docs/install_old.md

```diff
@@ -86,7 +96,7 @@ activate hercules`).
 
 In the first terminal, run
 ```
-helics_broker -t zmq  -f 2 --loglevel="debug"
+helics_broker -f 2 --consoleloglevel=trace --loglevel=debug --local_port=$HELICS_PORT &
```

Comment:
The original code doesn't work for me. It throws an error (shown below).
![image](https://github.com/NREL/hercules/assets/34248707/04641275-2398-41d3-9b6f-af1c4b8a22d7)